### ISSUE TITLE
2.7.2 tag in github has version 2.7.1 in package.json

### ISSUE THREAD

**Issue by ankon** [Blizzard/node-rdkafka#663]
**Steps to Reproduce**

https://github.com/Blizzard/node-rdkafka/blob/v2.7.2/package.json#L3

**Additional context**

If I were to guess: The procedure for making releases isn't using `npm version`, but rather is a set of steps to execute manually, and so this can happen.

Right now I think the best way forward is to release a 2.7.3 (so that things stay "compatible" according to semver) with the correct version in `package.json`.

**Comment by devinfgao** [comment]
@webmakersteve , can you please fix this asap?

**Comment by liweifeng2** [comment]
One host fix is blocked by this. Can someone take a look?

**Comment by shadowfool** [comment]
2.7.0 seems like a safe build to pin against if anyone else is having issues at the moment

**Comment by mafischer** [comment]
I'm seeing this issue with `package-lock.json`. The `2.7.1-2` version is installed correctly, but the `package-lock.json` shows version '2.7.2':

```
    "node-rdkafka": {
      "version": "2.7.2",
      "resolved": "https://registry.npmjs.org/node-rdkafka/-/node-rdkafka-2.7.1-2.tgz",
      "integrity": "sha512-sSpKsAGVeIn5l7Z3wNAtIP122MPK5rqgzNuRIGGwo6Z73bEoRgbjJmLhHwS8zx/lCCFYZ2YGKEnG48Ztr73XDA==",
      "requires": {
        "bindings": "^1.3.1",
        "nan": "^2.14.0"
      }
    },
```

**Comment by rashid-test** [comment]
> 2.7.0 seems like a safe release to pin against if anyone else is having issues at the moment

Yes, but 2.7.1. 2.7.0 doesn't' work at all for me.

**Comment by francistm** [comment]
When running with `npm ci`, it will show ` ETARGET
npm ERR! notarget No matching version found for node-rdkafka@2.7.2` exception

**Comment by IdanAdar** [comment]
This is resolved in 2.7.4. Issue can be closed
